.github/workflows/deploy.yml

```diff
@@ -4,13 +4,21 @@ on:
     types:
       - published
 jobs:
-  build:
-    name: Build
+  publish:
+    name: Publish
     if: ${{ !github.event.release.prerelease && github.event.release.target_commitish == 'master' }}
```

Comment:
Same question as the one on `on` can the branch not be specified in the `on` block?